homeassistant/components/imou_life/quality_scale.yaml

```diff
@@ -0,0 +1,75 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: Integration does not register custom actions.
+  appropriate-polling: done
+  brands:
```

Comment:
The brand is imou. Do you have separate brand for Imou life? If not I'd recommend to rename this integration to `imou` and put the brand assets under this name. This integration already supports multiple classes of device. If ever you create another imou integration, you can create an alias.
But in any case you still have to put assets in the brands repo so that the documentation could be rendered with the logo.